src/pages/ServiceList/__tests__/ItemDescription.test.tsx

```diff
@@ -24,15 +24,14 @@ describe('ItemDescription', () => {
     };
   });
 
-  it('should render with promise resolving', done => {
+  it('should render with promise resolving', () => {
     const wrapper = shallow(<ItemDescription item={item} />);
     expect(wrapper.text()).toBe('');
 
     resolver(health);
-    item.healthPromise.then(() => {
+    return new Promise(r => setImmediate(r)).then(() => {
```

Comment:
I'm not familiar with this... is returning a promise a similar thing than calling `done()` , with a guarantee that it's going to be evaluated?

Reply:
It's similar. Read about it here: https://jestjs.io/docs/en/asynchronous#promises

I changed this because it was the only way I could clear the "promise stack" before the assertions. Otherwise `expect`s were being evaluated before the component rendered (because of the internal cancelable promise).